src/Paulus/Exception/Http/Standard/HttpExceptionFactory.php

```diff
@@ -0,0 +1,61 @@
+<?php
+/**
+ * Paulus - A PHP micro-framework for creating RESTful services
+ *
+ * @author      Trevor Suarez (Rican7)
+ * @author      Mark Costello (mcos)
+ * @copyright   2015 Trevor Suarez
+ * @link        https://github.com/Rican7/Paulus
+ * @license     https://github.com/Rican7/Paulus/blob/master/LICENSE
+ * @version     2.0.0
+ */
+
+namespace Paulus\Exception\Http\Standard;
+
+use Exception;
+
+/**
+ * HttpExceptionFactory
+ *
+ * Provides the means to create Api Exceptions using minimal other information
+ *
+ * @package Paulus\Exception\Http\Standard
+ */
+class HttpExceptionFactory
+{
+
+    /**
+     * Create an HTTP exception using a HTTP status code.
+     *
+     * @param int $code	 The HTTP status code
+     * @param string	 $message
+     * @param Exception  $previous
```

Comment:
It looks like you indented with a `tab` character. Gotta use soft-tabs (spaces) here, a la PSR-2.